packages/cloud/deployment_details/deployment_details.tsx

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import React from 'react';
+
+import {
+  EuiForm,
+  EuiFormRow,
+  EuiFieldText,
+  EuiCopy,
+  EuiButtonIcon,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLink,
+  EuiButtonEmpty,
+  EuiSpacer,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { useDeploymentDetails } from './services';
+
+export const DeploymentDetails = ({ closeModal }: { closeModal?: () => void }) => {
+  const { cloudId, elasticsearchUrl, managementUrl, learnMoreUrl, navigateToUrl } =
+    useDeploymentDetails();
+  const isInsideModal = !!closeModal;
+
+  if (!cloudId) {
+    return null;
+  }
+
+  return (
+    <>
+      <EuiForm component="div">
+        {/* Elastic endpoint */}
+        {elasticsearchUrl && (
+          <EuiFormRow
+            label={i18n.translate('cloud.deploymentDetails.elasticEndpointLabel', {
+              defaultMessage: 'Elastic endpoint',
+            })}
+            fullWidth
+          >
+            <EuiFlexGroup gutterSize="s">
+              <EuiFlexItem>
+                <EuiFieldText
+                  value={elasticsearchUrl}
+                  fullWidth
+                  disabled
+                  data-test-subj="deploymentDetailsEsEndpoint"
+                />
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiCopy textToCopy={elasticsearchUrl}>
+                  {(copy) => (
+                    <EuiButtonIcon
+                      onClick={copy}
+                      iconType="copyClipboard"
+                      display="base"
+                      size="m"
+                    />
+                  )}
+                </EuiCopy>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFormRow>
```

Comment:
nit, would be gucci if this was one line:

```suggestion
       <DeploymentDetailsEsInput />
```

Reply:
And the next input the same.

Reply:
Good point 👍 
Addressed in https://github.com/elastic/kibana/pull/167122/commits/9d4628189d0a4d6134d6ed4e5eace36e05d222d7